Kernel/Bus/USB/USBConfiguration.cpp

```diff
@@ -55,7 +58,7 @@ ErrorOr<void> USBConfiguration::enumerate_interfaces()
                 raw_endpoint_descriptor_offset += sizeof(USBHIDDescriptor); // Skip the HID descriptor (this was worked out via buffer inspection)
 
             USBEndpointDescriptor endpoint_descriptor;
-            memcpy(&endpoint_descriptor, raw_endpoint_descriptor_offset, sizeof(USBEndpointDescriptor));
+            AK::TypedTransfer<u8>::copy(reinterpret_cast<u8*>(&endpoint_descriptor), raw_endpoint_descriptor_offset, sizeof(USBEndpointDescriptor));
```

Comment:
If this is needed I'd put it in a separate commit with a nice explanation why this can't/shouldn't be a memcpy

Reply:
Please don't use TypedTransfer in normal code. it's meant as implementation details of AK containers